extensions/objects/models/objects_test.py

```diff
@@ -453,7 +453,6 @@ def test_default_values_for_objects_are_valid(self):
                 self.assertEqual(
                     member.normalize(member.default_value),
                     member.default_value)
-
```

Comment:
Don't modify this file just for one line.

Reply:
No need to do that now.